### PR TITLE
Fix foxy install 

### PR DESCRIPTION
**Pull request by kids-return**
Fixed foxy install

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,25 +34,22 @@
...
-        "yiisoft/cache": "dev-master",
         "foxy/foxy": "^1.0",
-        "phpunit/phpunit": "^7.3",
+        "yiisoft/di": "^3.0@dev",
+        "yiisoft/log": "^3.0@dev",
+        "yiisoft/cache": "^3.0@dev",
         "yiisoft/yii-web": "^3.0@dev",
-        "hiqdev/composer-config-plugin": "^1.0@dev"
+        "yiisoft/yii-jquery": "^3.0@dev",
+        "yiisoft/yii-captcha": "^3.0@dev",
```
Comment (machour) [inline composer.json]: Why captcha?

Comment (kids-return) [reply composer.json]: Reference:https://github.com/yiisoft/yii-bootstrap4/blob/490cd1a7cfd8079ac6edba68f33519206611948d/composer.json#L50
How about optimization after that?

Comment (machour) [reply composer.json]: In bootstrap4, tests use Captcha.

In this repository it's not used, so you are adding it for nothing, please remove.

Comment (kids-return) [reply composer.json]: done.

**Comment by machour** [comment]
And now tests are passing, thank you :)

**Pull request merged**